lib/taskana-core/src/test/java/acceptance/task/QueryTasksWithSortingAccTest.java

```diff
@@ -98,6 +98,42 @@ void testSortByTaskIdAsc() {
     }
   }
 
+  @WithAccessId(user = "admin")
+  @Test
+  void testSortByWorkbasketName() {
+    TaskService taskService = taskanaEngine.getTaskService();
+    List<TaskSummary> results = taskService.createTaskQuery().orderByWorkbasketName(asc).list();
+
+    // test is only valid with at least 2 results
+    assertThat(
+            results.stream()
+                .map(t -> t.getWorkbasketSummary().getName())
+                .distinct()
+                .collect(Collectors.toList()))
+        .hasSizeGreaterThan(2);
+
+    List<String> idsAsc =
+        results.stream()
+            .map(t -> t.getWorkbasketSummary().getName())
```

Comment:
according to our coding guidelines this is not ok.
Please use method references